src/aat/resources/features/F-066/F-066_Test_Data_Base.td.json

```diff
@@ -0,0 +1,27 @@
+{
+	"_guid_": "F-066_Test_Data_Base",
+	"productName": "CCD Data Store",
+	"operationName": "Retrieve a start event trigger by ID for dynamic display",
+	"method": "GET",
+	"uri": "/internal/cases/{cid}/event-triggers/{etid}",
+
+	"user": {
+		"_extends_": "Common_User_For_Request"
+	},
+
+	"request": {
+    "headers": {
+      "_extends_": "Common_Request_Headers",
+      "experimental": true
+    },
+    "pathVariables": {
+      "cid": "15737516488407031",
```

Comment:
do we need to hardcode the case id here or could it be dynamic? otherwise the case creation step is redundant

Reply:
Yes, I discussed it with Mutlu and as per his suggestions the hardcoded reference number is required in such scenarios in order to avoid creating new case every time even for negative scenarios where case reference number won't be checked.